src/index.js

```diff
@@ -30,6 +30,11 @@ const languageWithoutRegionCode = language.toLowerCase().split(/[_-]+/)[0];
 // Try full locale, try locale without region code, fallback to 'en'
 const messages = localeData[languageWithoutRegionCode] || localeData[language] || localeData.en;
 
+if (configuration.api.substr(0, 5) !== 'https') {
```

Comment:
Doesn't make any sense. Either show an error and don't render the page or redirect to https. But don't just console.error here if https isn't used.

Reply:
Removed it. I think this happend during the move from GitLab to GitHub.